apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableJobPostings.tsx

```diff
@@ -255,24 +254,16 @@ function ModalForm({
 
   let initialFormValues
   if (tpJobListingId) {
-    // Editing existing job posting
-    initialFormValues = pick(
-      jobListing,
-      'title',
-      'location',
-      'summary',
-      'relatesToPositions',
-      'idealTechnicalSkills',
-      'employmentType',
-      'languageRequirements',
-      'isRemotePossible',
-      'federalState',
-      'salaryRange',
-      'contactFirstName',
-      'contactLastName',
-      'contactPhoneNumber',
-      'contactEmailAddress'
-    )
+    // This is a bit of a hack, but it works for now.
+    // We need to make sure initialFormValues have string
+    // contact info otherwise Yup validation errors are weird.
+    initialFormValues = {
+      ...jobListing,
+      contactFirstName: jobListing?.contactFirstName || '',
+      contactLastName: jobListing?.contactLastName || '',
+      contactEmailAddress: jobListing?.contactEmailAddress || '',
+      contactPhoneNumber: jobListing?.contactPhoneNumber || '',
+    }
```

Comment:
It's weird, but the error message was weird only for `contactPhoneNumber` field.